src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return String.valueOf(Colors.values()[index]);
```

Comment:
Use the name() method to get the String representation of the enum constant instead of String.valueOf(), which is less efficient and can be problematic if toString() is overridden.